app/observers/project_observer.rb

```diff
@@ -66,6 +66,29 @@ def from_approved_to_online(project)
       audited_user_moip_login: project.user.moip_login,
       audited_user_phone_number: project.user.phone_number
     })
+    AuditedAccount.create(
```

Comment:
@adrianob I think that we don't need this, you should use the form of dashboard_user_settings to save / create the audited_account though accept_nested_attributes... what u think?